src/google/protobuf/unittest_proto3.proto

```diff
@@ -140,6 +140,11 @@ message TestAllTypes {
     string oneof_string = 113;
     bytes oneof_bytes = 114;
   }
+  
+  // Sample maps
```

Comment:
I didn't notice that, thanks! Can you try to make it similar with map_unittest.proto? Could use NestedMessage instead for proto2 messages like your current code.

I understand there's no need to test all the types like C++. Just put the definition in .proto file and comments why not test them is good enough.